dungeon/src/contrib/hud/UIUtils.java

```diff
@@ -64,30 +64,74 @@ public static void center(final Actor actor) {
   }
 
   /**
-   * Creates line breaks after a word once a certain character count is reached.
+   * This is a simple implementation of formatting a text with soft word wrap.
+   *
+   * <p>Characteristics:
+   *
+   * <ul>
+   *   <li>Every space character (i.e. {@code " "} or {@code "\n"}, etc.) that occurs at least once
+   *       should be replaced by exactly one space character.
+   *   <li>A line of text should not be longer than {@code MAX_ROW_LENGTH} (currently 40)
+   *       characters.
+   *   <li>Long words at the end of a line should be inserted into the next line (soft word wrap).
```

Comment:
oha. das wäre also ein "soft wrap". aber was ist nun mit wörtern, die länger als eine zeilenlänge sind? die kannst du ja nicht einfach in die nächste zeile schieben, weil sie dort auch zu lang sind? wenn ich das richtig sehe, splittest du diese wörter dann doch. das gehört aber hier mit in die beschreibung.